<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertLinkDialog</class>
 <widget class="QDialog" name="InsertLinkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вставить ссылку</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblText">
     <property name="text">
      <string>Текст ссылки</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edtText"/>
   </item>
   <item>
    <widget class="QLabel" name="lblURL">
     <property name="text">
      <string>Адрес ссылки</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edtURL">
     <property name="placeholderText">
      <string>http://example.com/</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InsertLinkDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InsertLinkDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
